lib/iris/fileformats/grib/_load_convert.py

```diff
@@ -1662,6 +1662,19 @@ def statistical_cell_method(section):
     return cell_method
 
 
+def ensemble_identifier(section):
```

Comment:
This is a new function that does not have an associated test - I recommend a new test module in `lib/iris/tests/units/fileformats/grib/load_convert`.